modules/unsupported/geopkg/src/main/java/org/geotools/geopkg/GeoPackage.java

```diff
@@ -436,7 +415,20 @@ public void addCRS(int srid, String srsName, String organization, int organizati
      * 
      */
     public void addCRS(CoordinateReferenceSystem crs, String auth, int srid) throws IOException {
-        this.addCRS(srid, auth + ":" + srid, auth, srid, crs.toWKT(), auth + ":" + srid);
+
+        Connection cx;
+        try {
```

Comment:
No need to change right now, but a suggestion for the future: connection is AutoCloseable, a single try-with-resources could have been used here.